src/js/utils/__tests__/FormUtil-test.js

```diff
@@ -0,0 +1,79 @@
+jest.dontMock('../FormUtil');
+
+var FormUtil = require('../FormUtil');
+
+describe('FormUtil', function () {
+  describe('#getMultipleFields', function () {
+    describe('one property', function () {
+      beforeEach(function () {
+        this.result = FormUtil.getMultipleFields('uid', 2, {
+          name: 'uid'
+        });
+      });
+
+      it('returns the correct amount of fields', function () {
+        expect(this.result.length).toEqual(2);
+      });
+
+      it('returns the correct names for fields', function () {
+        expect(this.result[0].name).toEqual('uid[0]');
```

Comment:
I was also wondering if we really need the the prop to group things. Wouldn't the index be enough?